karma.conf.js

```diff
@@ -0,0 +1,62 @@
+// Karma configuration
+// Generated on Tue Dec 08 2015 23:01:01 GMT-0800 (Pacific Standard Time)
+
+module.exports = function (config) {
+  config.set({
+
+    // base path that will be used to resolve all patterns (eg. files, exclude)
+    basePath: '',
+
+    // frameworks to use
+    // available frameworks: https://npmjs.org/browse/keyword/karma-adapter
+    frameworks: ['browserify', 'jasmine'],
+
+    // list of files / patterns to load in the browser
+    files: [
+      'spec/helpers/marble-testing.js',
+      'spec/helpers/test-helper.js',
+      'spec/**/*-spec.js'
+    ],
+
+    // list of files to exclude
+    exclude: [
+      'spec/observables/from-promise-spec.js' //'should globally throw unhandled errors' fails
```

Comment:
I predicted that test would bring some headaches (I wrote that test in order to increase test coverage). One option is we can move just that test to `spec/observables/fromPromise-node-spec.js`.